src/components/Header/HeaderContainer.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import axios from "axios";
+
+import Header from "./Header";
+import { connect } from "react-redux";
```

Comment:
Consider fixing the import grouping here.

Reply:
done